src/handle-data.js

```diff
@@ -15,12 +15,20 @@ export function getColumnOrder({ qColumnOrder, qDimensionInfo, qMeasureInfo }) {
   return qColumnOrder?.length === columnsLength ? qColumnOrder : [...Array(columnsLength).keys()];
 }
 
-export function getColumnInfo(qHyperCube, colIndex) {
+export function getColumnInfo(qHyperCube, colIndex, layout) {
```

Comment:
`qHyperCube` exists on `layout`, only pass layout

Reply:
Ah, you're right! It was there,  I see I missed to remove it after adding layout